### ISSUE TITLE
Change the background color of the form in the contact section

### ISSUE THREAD

**Issue by YoshithaRathnayake** [YoshithaRathnayake/NEW-YOSHITHA-WEBSITE#165]
- Currently, the background color of the form in the contact section is different than the default section background color
- Change the background color of the form to the background color in the default section `second-section`

![image](https://user-images.githubusercontent.com/97069900/206942712-f9ac8e57-981d-482a-861c-132524e11536.png)

**Comment by Rishav1707** [comment]
Shall I work on this issue?? @YoshithaRathnayake

**Comment by YoshithaRathnayake** [comment]
> Shall I work on this issue?? @YoshithaRathnayake

Sure

**Comment by Rishav1707** [comment]
![Screenshot 2022-12-12 182600](https://user-images.githubusercontent.com/97666287/207051472-b3d0f4b1-1f30-4d69-9955-df8d360002cf.png)

How can I access this css property which is marked as important.
This CSS property is in utilities class.
I am not able to change this property. 
@YoshithaRathnayake

**Comment by Rishav1707** [comment]
I am trying to change but it does not apply to the particular class name.

**Comment by YoshithaRathnayake** [comment]
> ![Screenshot 2022-12-12 182600](https://user-images.githubusercontent.com/97666287/207051472-b3d0f4b1-1f30-4d69-9955-df8d360002cf.png)
> 
> How can I access this css property which is marked as important. This CSS property is in utilities class. I am not able to change this property. @YoshithaRathnayake

No no you have to find the css class, there is a css class for that form. Add the codes to it @Rishav1707

**Comment by Rishav1707** [comment]
ok let me try again to find that CSS class.

**Comment by YoshithaRathnayake** [comment]
https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/blob/99478ac65f3ce96bac21aaa7658347798bfff94a/dark/assets/css/styles.css#L1094-L1100

Here the class
Aren't you using vs code? @YoshithaRathnayake

**Comment by Rishav1707** [comment]
yaah but when I apply the background color property in this class it does not apply to the website.
It neglected.

**Comment by YoshithaRathnayake** [comment]
> yaah but when I apply the background color property in this class it does not apply to the website. It neglected.

You have to remove the bg-dark in the bootstrap class first in the element after that do the changes @Rishav1707

**Comment by YoshithaRathnayake** [comment]
@AlizayAyesha
